spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -143,4 +156,20 @@ def action_is_shown(key)
       expect(page).to have_link(action.guidance_link_text, href: action.guidance_url)
     end
   end
+
+  def and_the_tourism_link_should_have_tracking_analyitics
+    action = BrexitChecker::Action.find_by_id("T063")
+    expect(page).to have_css(".govuk-link[href='#{action.guidance_url}'][data-track-action='Your business or organisation - 1.2 - Guidance']")
+
```

Comment:
Can we also check for `data-track-category="brexit-checker-results"` and `data-track-label=[the url]`?

Reply:
Are we able to check the tracking on the action link too, if this action has one?